api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -284,6 +284,10 @@ private void loadAppsAndExtensions() {
             enablePrimaryCare();
         }
 
+        if (config.isComponentEnabled(Components.MEXICO_CLINIC)) {
```

Comment:
@brandones currently components generally are grouped by functional areas... would be nice if we could turn this into a functional component (or use an existing one--outpatient consult?) and then potentially configure that component based on country.  Though if you think will just make it more confusing, I could possibly be persuaded this just having a Mexico Clinic is a better way... :)